multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -528,6 +530,7 @@ def fit(
         presets: Optional[str] = None,
         config: Optional[dict] = None,
         tuning_data: Optional[Union[pd.DataFrame, str]] = None,
+        max_tuning_num: Optional[int] = None,
```

Comment:
`max_tuning_num` may also sound like the number of validations to conduct.

Reply:
True, how about max_num_tuning_data